docs/_includes/ds6/select.html

```diff
@@ -11,6 +11,9 @@ <h3>Default select</h3>
                     <option value="item2">Option 2</option>
                     <option value="item3">Option 3</option>
                 </select>
+                <svg class="icon icon--chevron-down-bold" focusable="false" height="24" width="24">
```

Comment:
@seangates as we discussed, please add the documentation to mention that SVG definitions should be included, in order for this to work.